src/views/base_view.js

```diff
@@ -34,6 +37,11 @@ export default AmpersandView.extend({
     dom.hide(this.el);
   },
 
+  // Adjust the height of the iframe to match the height of the body.
+  adjustHeight () {
+    webChannel.sendAdjustHeight(document.body.offsetHeight);
```

Comment:
Yeah, let's definitely avoid styling body if at all possible.

I'm forever thinking of a future in which these same views are rendered in a regular webpage.